operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -191,12 +191,14 @@ spec:
 #      url: ""
 # **Tracing-specific settings:
 #  - Right now we only support Jaeger
+# enabled: When true, Kiali shows Jaeger and autodiscover
 # namespace: The Kubernetes namespace that holds the Tracing service (if empty, assumes the same value as deployment.namespace)
 # service: The Kubernetes service name for tracing. Kiali uses this to connect within the cluster to Jaeger.
 # url: The URL that Kiali uses when integrating with Tracing. This URL must be accessible to clients external to
 #      the cluster in order for the integration to work properly. If empty, an attempt to auto-discover it is made.
 #    ---
 #    tracing:
+#      enabled: "true"
```

Comment:
looking at the defaults/main.yml, the default is actually `true` not `"true"` (actual boolean, not string). So this should be:

```suggestion
#      enabled: true
```

Reply:
so display_link should be boolean too. I found it strange to see it